install.sh

```diff
@@ -31,42 +31,39 @@ tmp_dir="$tmp/install.sh.$$"
 # do_download URL DIR
 do_download() {
   echo "Downloading $1"
-  if [[ $(type -t wget) ]]; then
+  if [[ $(command -v wget) ]]; then
     wget -q -O "$2" "$1" >/dev/null
-  elif [[ $(type -t curl) ]]; then
+  elif [[ $(command -v curl) ]]; then
     curl --fail -sSL -o "$2" "$1" &>/dev/null || true
   else
     error "Could not find wget or curl"
     return 1
   fi
 }
 
-# get_latest_version URL
+# get_latest_version
 get_latest_version() {
-  versionloc=${tmp_dir}/summon.version
-  versionfile=$(do_download "${1}" "${versionloc}")
-  if [ -f "${versionloc}" ]; then
-    local version=$(cat ${versionloc} | grep -o -e "[[:digit:]].[[:digit:]]*.[[:digit:]]*")
-    echo "${version}"
+  if [[ $(command -v curl) ]]; then
+    curl --silent "https://api.github.com/repos/cyberark/summon/releases/latest" | # Get latest release from GitHub api
+    grep '"tag_name":' |                                            # Get tag line
+    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
+  else
```

Comment:
Some distros don't have `curl` iirc and most container won't have neither curl nor wget OOTB - we should try to use `wget` or `curl` depending on what's installed just like on L34.

```
$ docker run --rm -it alpine
/ # wget
BusyBox v1.28.4 (2018-07-17 15:21:40 UTC) multi-call binary.
...
/ # curl
/bin/sh: curl: not found
```

```
$ docker run --rm -it ubuntu
root@352476ef787b:/# curl
bash: curl: command not found
root@352476ef787b:/# wget
bash: wget: command not found
```

```
docker run --rm -it debian
root@ab56ab37cf83:/# wget
bash: wget: command not found
root@ab56ab37cf83:/# curl
bash: curl: command not found
```

Reply:
agreed and addressed :)

Reply:
agreed and addressed :)